src/stark/stark_commit.cairo

```diff
@@ -1 +1,114 @@
+use core::traits::TryInto;
+use core::array::ArrayTrait;
+use core::option::OptionTrait;
+use core::traits::Into;
+use cairo_verifier::channel::channel::ChannelTrait;
+use cairo_verifier::{
+    air::{
+        constants::{CONSTRAINT_DEGREE, N_CONSTRAINTS, N_INTERACTION_ELEMENTS, MASK_SIZE},
+        global_values::InteractionElements, public_input::PublicInput, traces::traces_commit,
+    },
+    channel::channel::Channel, common::powers_array::powers_array, domains::StarkDomains,
+    oods::OodsValues, fri::fri::fri_commit,
+    stark::{StarkUnsentCommitment, StarkConfig, StarkCommitment},
+    proof_of_work::proof_of_work::proof_of_work_commit, table_commitment::table_commit,
+    oods::verify_oods,
+};
 
+#[derive(Drop, Copy)]
```

Comment:
I think we should avoid this kind of structs lets just add comment in a code what the coefficients represent and omit this structs

Reply:
sure, fixed